be/src/exec/pipeline/exchange/sink_buffer.cpp

```diff
@@ -174,6 +182,7 @@ void SinkBuffer::cancel_one_sinker() {
 
 void SinkBuffer::_update_network_time(const TUniqueId& instance_id, const int64_t send_timestamp,
                                       const int64_t receive_timestamp) {
+    _last_receive_time = MonotonicNanos();
```

Comment:
the last request must contain eos, maybe we needn't update _last_receive_time every time.

Reply:
Emm, you cannot determine whether a result package is a eos or not. And seems unnecessary to add a struct to save the excess info